### PR TITLE
Applied fixes from FlintCI

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

File: src/DependencyInjection/Compiler/AdminMakerCompilerPass.php [hunk 1]
```diff
@@ -21,13 +23,13 @@
  */
 final class AdminMakerCompilerPass implements CompilerPassInterface
 {
-    const MANAGERS = [
+    public const MANAGERS = [
```
Comment (kunicmarko20) [inline src/DependencyInjection/Compiler/AdminMakerCompilerPass.php]: this can be private?

Comment (OskarStark) [reply src/DependencyInjection/Compiler/AdminMakerCompilerPass.php]: Yes it should be private 👍🏻

Comment (OskarStark) [reply src/DependencyInjection/Compiler/AdminMakerCompilerPass.php]: Or can we get it from the configuration somehow?

Comment (OskarStark) [reply src/DependencyInjection/Compiler/AdminMakerCompilerPass.php]: You can see it here: https://travis-ci.org/sonata-project/SonataAdminBundle/jobs/438008151

**Comment by OskarStark** [comment]
ah its another const :)

We need to rebase this branch, or can we simply close it and FlintCi will create a new PR?

**Comment by greg0ire** [comment]
Ah right, I was starting to think this was a weird php-cs-fixer bug! Do whichever solution you prefer :)

**Comment by OskarStark** [comment]
I will close this one here

**Pull request closed without merging**